statvar_imports/india_ndap/IndiaNSS_HealthAilments/IndiaNSS_HealthAilments_place_resolved.csv

```diff
@@ -6,7 +6,8 @@ Assam,wikidataId/Q1164
 Bihar,wikidataId/Q1165
 Chandigarh,wikidataId/Q43433
 Chhattisgarh,wikidataId/Q1168
-The Dadra and Nagar Haveli and Daman and Diu,wikidataId/Q77997266
+Dadra & Nagar Haveli,wikidataId/Q46107
```

Comment:
Is the source now coming with the place split into Dadra & Nagar Haveli and Daman & Diu ? Either case we can retain the combined entry as well. Since the Union territory is the combined one, high chance the data comes with the combined name in future.